### ISSUE TITLE
Urgent Help - flutter Xcode error, Multiple commands produce

### ISSUE THREAD

**Issue by NTMS2017** [flutter/flutter#31259]
I build my apk and place to google play console and had zero problem. 

Unfortunately when I try to do same for App Store I get a below error. I follow flutter docs on iOS deployment and error was in "Finally, create build archive and upload it to App Store Connect" section. When I try to select Product>Archive to produce a build archive get the below error. Currently I am connected as team member for company developer account.

I need to upload for test flight and failed. I need a urgent help. 

Thanks

```
Showing Recent Messages

Prepare build
note: Using new build systemnote: Planning buildnote: Constructing build description

Build system information
error: Multiple commands produce '/Users/*****/Library/Developer/Xcode/DerivedData/Runner-aefhndlkfxejntgefobrayuopjge/ArchiveIntermediates/Runner/InstallationBuildProductsLocation/Applications/Runner.app/Frameworks/Flutter.framework':
1) Target 'Runner' has copy command from '/Users/******/NTMS/capital_cep/ios/Flutter/Flutter.framework' to '/Users/********/Library/Developer/Xcode/DerivedData/Runner-aefhndlkfxejntgefobrayuopjge/ArchiveIntermediates/Runner/InstallationBuildProductsLocation/Applications/Runner.app/Frameworks/Flutter.framework'
2) That command depends on command in Target 'Runner': script phase “[CP] Embed Pods Frameworks”

Build system information
warning: The iOS deployment target is set to 6.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'Reachability')
Build system information
warning: ignoring duplicated output file: '/Users/*****/Library/Developer/Xcode/DerivedData/Runner-aefhndlkfxejntgefobrayuopjge/ArchiveIntermediates/Runner/InstallationBuildProductsLocation/Applications/Runner.app/Frameworks/Flutter.framework' in shell script build phase '[CP] Embed Pods Frameworks'. This warning represents an extremely serious project misconfiguration and will likely cause some shell scripts in your project to be skipped entirely, leading to other build failures or missing files in the build directory. This will be a hard error in the future. (in target 'Runner')


Build failed    18/04/2019, 16:18    1.1 seconds

```

**Comment by udit-wynk** [comment]
I am facing the exact same issue with my flutter application. Please help

**Comment by udit-wynk** [comment]
flutter doctor -v
[✓] Flutter (Channel beta, v1.2.1, on Mac OS X 10.14.3 18D109, locale en-IN)
    • Flutter version 1.2.1 at /Users/a1a22rd1/flutter
    • Framework revision 8661d8aecd (3 months ago), 2019-02-14 19:19:53 -0800
    • Engine revision 3757390fa4
    • Dart version 2.1.2 (build 2.1.2-dev.0.0 0a7dcf17eb)

[!] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/a1a22rd1/Library/Android/sdk
    • Android NDK at /Users/a1a22rd1/Library/Android/sdk/ndk-bundle
    • Platform android-28, build-tools 28.0.3
    • ANDROID_HOME = /Users/a1a22rd1/Library/Android/sdk
    • ANDROID_SDK_ROOT = /Users/a1a22rd1/Library/Android/sdk
    • Java binary at: /Applications/Android Studio
      v3.0.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1343-b01)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor
      --android-licenses

[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • ios-deploy 1.9.4
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio v3.0.app/Contents
    • Flutter plugin version 34.0.2
    • Dart plugin version 183.5901
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1343-b01)

[!] IntelliJ IDEA Community Edition (version 2018.1.4)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.io/intellij-setup/#installing-the-plugins

[✓] VS Code (version 1.33.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.25.0

[!] Connected device
    ! No devices available

! Doctor found issues in 3 categories.

**Comment by udit-wynk** [comment]
Update:

I was able to fix the issue by updating Xcode to 10.2.1.
In the process had to update my dependencies too (pod update)

**Comment by jmagman** [comment]
This is https://github.com/flutter/flutter/issues/20685.
